### DISCUSSION TITLE
21.2.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1290] [General]
## Releases Notes for 21.2.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/draw.io-21.2.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/draw.io-21.2.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/draw.io-universal-21.2.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/drawio-amd64-21.2.1.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/drawio-x86_64-21.2.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.1/drawio-x86_64-21.2.1.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Performance improvements for large diagrams
- Updates to [draw.io core 21.2.1](https://github.com/jgraph/drawio/blob/v21.2.1/ChangeLog). All changes from 21.1.3 to 21.2.1 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.2.1'>21.2.1</a>.</em>

**Comment by ToniestTony** [comment]
I have a very large diagram that used to lag so much that I couldn't keep working on it, but this update fixed the lag ! This is super appreciated, thanks a lot
